ml-data-hub-plugin/src/main/groovy/com/marklogic/gradle/DataHubPlugin.groovy

```diff
@@ -139,6 +139,9 @@ class DataHubPlugin implements Plugin<Project> {
                 "Requires -Pconfirm=true to be set so this isn't accidentally executed.")
         project.task("hubClearUserArtifacts", type: ClearUserArtifactsTask, group: developGroup,
             description: "Clears all user artifacts in the staging, final databases. Requires -Pconfirm=true to be set so this isn't accidentally executed.")
+        project.task("hubClearUserModules", type: ClearUserModulesTask, group: developGroup,
+            description: "Clears user modules in the modules database, only leaving only the modules " +
```

Comment:
Couple typos - do "only leaving the modules that are loaded when Data Hub is installed."